src/common/meta/src/ddl/alter_table.rs

```diff
@@ -335,13 +353,24 @@ impl AlterTableProcedure {
     }
 
     fn lock_key_inner(&self) -> Vec<String> {
+        let mut lock_key = vec![];
+
+        if let Some(physical_table_name) = self.data.physical_table_name() {
+            let physical_table_key = common_catalog::format_full_table_name(
+                &physical_table_name.catalog_name,
+                &physical_table_name.schema_name,
+                &physical_table_name.table_name,
+            );
+            lock_key.push(physical_table_key);
```

Comment:
Can we use the physical table id as lockkey (convert it to String of course)? I'm aware physical table name has little chance to be renamed, but it's better to use a hard invariant.

Reply:
We use the table name (Failover and Migration) in other procedures; it is necessary to ensure mutual exclusivity with these.

Reply:
Rename Physical table name is not allowed.

Reply:
> We use the table name (Failover and Migration) in other procedures; it is necessary to ensure mutual exclusivity with these.

BTW, I will refactor these keys and add more tests recently.